components/infra-proxy-service/server/orgs.go

```diff
@@ -91,25 +91,20 @@ func (s *Server) GetOrg(ctx context.Context, req *request.GetOrg) (*response.Get
 
 	UUID, err := uuid.FromString(req.Id)
 	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "invalid org id: %s", err.Error())
+		return nil, status.Errorf(codes.InvalidArgument, "invalid org ID: %s", err.Error())
 	}
 
 	org, err := s.service.Storage.GetOrg(ctx, UUID)
 	if err != nil {
 		return nil, service.ParseStorageError(err, req.Id, "org")
 	}
 
-	secret, err := s.service.Secrets.Read(ctx, &secrets.Id{Id: org.AdminKey})
-	if err != nil {
-		return nil, err
-	}
-
 	return &response.GetOrg{
 		Org: &response.Org{
 			Id:        org.ID.String(),
 			Name:      org.Name,
 			AdminUser: org.AdminUser,
-			AdminKey:  GetOrgAdminKeyFrom(secret),
+			AdminKey:  org.AdminKey,
```

Comment:
Sending Secret admin key ID for interservice use.

Reply:
would a user *ever* want to update the key for an org?
what if they create an org with a bad key?  they could delete and recreate, but it seems like we could still allow for update. 
in the node integrations we have a similar flow - user inputs the secret data when creating the integration, and we save it as a secret with the secret service. if the user later wants to edit those credentials, we allow them to do so, while still obfuscating the credentials on object read.
take a look at:
https://github.com/chef/automate/blob/master/components/nodemanager-service/server/manager/server.go#L83-L88
https://github.com/chef/automate/blob/master/components/nodemanager-service/server/manager/server.go#L119-L156

Reply:
Returning AdminKey(saved the secrets-service reference ID) here, only meant for any actional purpose. now we going to store and manage the Admin key using `secrets-service` this might be worth renaming it to `credetial_id`?

Reply:
> would a user ever want to update the key for an org?

I would possibly say yes because adding/connecting chef servers with bad info might be possible as of now we don't have any provision to validate the given info on the fly until and unless the user will go to access data from chef servers.


> what if they create an org with a bad key? they could delete and recreate, but it seems like we could still allow for update.

Yes, this could work as well to delete & re-create if anything goes wrong.

Reply:
spoke with btm about this:
"yes. it's possible that keys could get rotated due to a security event and you would need to change the key. it'd be annoying to have to delete everything and recreate it to change the key."

so let's 
1) still let users updated the admin key from org update
2) not show them the key they already input, bc that's leaking the credential

do you need anything from me or ux to make that happen?

we also discussed sharing credentials -- we shouldn't encourage it - each org should have its own admin user/key

Reply:
Thanks, @vjeffrey for the confirmation...

> do you need anything from me or ux to make that happen?

Will check for possible way, Surely will inform you thankyou.